rules.d/50-udev-default.rules.in

```diff
@@ -41,6 +41,7 @@ SUBSYSTEM=="drm", KERNEL=="renderD*", GROUP="render", MODE="{{GROUP_RENDER_MODE}
 SUBSYSTEM=="kfd", GROUP="render", MODE="{{GROUP_RENDER_MODE}}"
 
 SUBSYSTEM=="misc", KERNEL=="sgx_enclave", GROUP="sgx", MODE="0660"
+SUBSYSTEM=="misc", KERNEL=="sgx_vepc", GROUP="sgx", MODE="0660"
```

Comment:
Not familiar with this, but isn't this sgx_epc?

Reply:
It is sgx_vepc. It is used by QEMU to allocate a chunk of EPC for KVM.

Reply:
https://wiki.qemu.org/ChangeLog/6.2#KVM

Reply:
Thank you for the reference. Then, please fix the commit message.

Reply:
Uh oh my commit message is wrong! I'll fix it.

Reply:
I fixed the commit message: s/sgx_epc/sgx_vepc/.

Reply:
Thank you!